db/schema.sql

```diff
@@ -804,3 +805,38 @@ create table host_deleted (
     primary key (hostid)
 ) engine=InnoDB;
 
+create table consent (
+    id                      integer         not null auto_increment,
+    userid                  integer         not null,
+    consent_type_id         integer         not null,
+    consent_time            integer         not null,
+    consent_flag            tinyint         not null,
+    consent_not_required    tinyint         not null,
+    source                  varchar(255)    not null,
+    primary key (id)
+) engine=InnoDB;
+
+create table consent_type (
+    id                      integer         not null auto_increment,
+    shortname               varchar(255)    not null,
+    description             varchar(255)    not null,
+    enabled                 integer         not null,
+    protect                 integer         not null,
```

Comment:
protect describes the how the application uses this field.  In your design document you describe the field as "project specific".  Would it be better to name it project_specific rather than 'protect'?   That makes its meaning clear without having to resort to the design document.

Reply:
Fixed in b44a8c8 and 992cece